### PR TITLE
Add ovito

### PR DESCRIPTION
**Pull request by jan-janssen**
OVITO is a scientific visualization and analysis software for atomistic and particle simulation data.

### CODE REVIEW

File: recipes/ovito/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+mkdir build
+cd build
+if [ ${PY3K} = 1 ]; then 
+	cmake -DOVITO_BUILD_PLUGIN_MESH=ON -DOVITO_BUILD_PLUGIN_PARTICLES=ON -DOVITO_BUILD_PLUGIN_TACHYON=OFF -DOVITO_BUILD_PLUGIN_CRYSTALANALYSIS=OFF -DOVITO_BUILD_PLUGIN_NETCDFPLUGIN=OFF -DOVITO_BUILD_PLUGIN_POVRAY=OFF -DOVITO_BUILD_PLUGIN_OPENBABELPLUGIN=OFF -DOVITO_BUILD_PLUGIN_VR=OFF -DOVITO_BUILD_PLUGIN_CORRELATION=ON -DOVITO_BUILD_PLUGIN_VOROTOP=ON -DOVITO_BUILD_GUI=OFF -DOVITO_BUILD_DOCUMENTATION=OFF -DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE=${PREFIX}/bin/python -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER}m -DPYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m.so -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
```
Comment (mariusvniekerk) [inline recipes/ovito/build.sh]: can you split this line please.

**Comment by jan-janssen** [comment]
@mariusvniekerk Thank you for your feedback - I updated the recipe accordingly.

File: recipes/ovito/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - qscintilla2
+    - sphinx
+    - numpy
```
Comment (mariusvniekerk) [inline recipes/ovito/meta.yaml]: You likely want to move the various python libs to has as well, but i don't view that as blocking

**Comment by jan-janssen** [comment]
@mariusvniekerk - I moved the Python requirements to the host section

**Pull request merged**